docs/book/src/developer/release-support-guidelines.md

```diff
@@ -34,8 +34,10 @@ standard [backport policy](https://github.com/kubernetes-sigs/cluster-api/blob/m
 
 | Minor Release | API Version | Supported Until                                    |
 |---------------|-------------|----------------------------------------------------|
-| v0.7.x        | **v1beta2** | when v0.9.0 will be released                       |
-| v0.6.x        | **v1beta2** | when v0.8.0 will be released                       |
+| v0.9.x        | **v1beta2** | when v0.11.0 will be released                       |
```

Comment:
why do we need 0.9 entry in 0.8 release branch?

Reply:
will fix it.